Add personal description search helper text and tooltip

louh opened the pull request:
[Link to issue.](https://github.com/usdoj-crt/crt-portal-management/issues/1137)

## What does this change?

Adds instruction text and a helper tooltip to the personal description search field so that intake users understand that AND or OR keywords are possible in a search filter. The tooltip uses the stock USWDS tooltip component and can be opened with a keyboard. It would still be good to get a review on how different screen readers behave when interacting with the tooltip.

To avoid confusion, the inputs for CRT summary and Incident location name have been moved alongside the Prison type dropdown.

Additionally, the "Apply filters" button has moved to the right side on both the filter view and dashboard view, to more closely align with design mockups. Note: while the the design mockups use a black background on tooltips, the tooltip currently uses the primary dark color of the theme throughout the site. After reviewing with @chinelo-ikejimba we have left that original theme color the same.

## Screenshots (for front-end PR):

Helper text and help icon:

<img width="1359" alt="Screen Shot 2021-12-06 at 1 18 38 PM" src="https://user-images.githubusercontent.com/2553268/144901902-0e6ac88b-5fa6-404a-ad97-5b62f39ba1cd.png">

Help icon expanded with tooltip:

![Screenshot 2021-12-06 at 13-21-36 Contact the Civil Rights Division Department of Justice](https://user-images.githubusercontent.com/2553268/144901927-d3231e2d-8d4e-4c50-8cef-482b61fded72.png)

## Checklist:

### Author

+ [x] If this is a story, run locally and check to make sure all Acceptance Criteria are met. (If any criteria are unclear, ask about them.).
+ [x] Check for, document, and establish a testing plan for any behavior that may vary across environments or is otherwise difficult to test.
+ [x] Check for [accessibility](https://github.com/usdoj-crt/crt-portal/blob/develop/docs/a11y_plan.md).
+ [x] [Tests pass](https://github.com/USDOJ/crt-portal/#tests).

### Reviewer

+ [ ] If this is a story, run locally and check to make sure all Acceptance Criteria are met. (If any criteria are unclear, ask about them.).
+ [ ] Check for any behavior that may vary across environments or is difficult to test, and ensure that it is well-understood, documented, and that there is a testing plan in place.
+ [ ] Re-check for [accessibility](https://github.com/usdoj-crt/crt-portal/blob/develop/docs/a11y_plan.md).
+ [ ] [Tests pass](https://github.com/USDOJ/crt-portal/#tests).

## Notes for reviewer:

See [PR instructions doc](https://github.com/usdoj/crt-portal/blob/master/docs/pull_requests.md) for full pull request review instructions.

ilodidoj commented:
@louh rendering of tooltip is not consistent due to large amount of text and paragraph format.  I think modal will be good approach than a tool tip for consistent rendering. I checked this only on google chrome browser on mac.

louh commented:
I've updated this PR but also wanted to make sure I recorded what the observed problem was, and how we fixed it.

As @ilodidoj discovered, if the tooltip icon is too close to the top or bottom edge of the window, the tooltip takes on an unexpected size, and text starts to overflow outside of it:

<img width="668" alt="Screen Shot 2021-12-08 at 12 39 30 PM" src="https://user-images.githubusercontent.com/2553268/145256704-d712fcf9-ebc8-4c4e-b3a0-f5eb9bb486e9.png">

We've now fixed the issue. The tooltip will retain its expected size:

<img width="449" alt="Screen Shot 2021-12-08 at 11 57 30 AM" src="https://user-images.githubusercontent.com/2553268/145257015-0b41bf00-0600-45a1-a4e1-e47cbd4bb759.png">

Note that the tooltip still isn't positioned fully within the viewport window in these cases. This is a limitation of the USWDS tooltip component's behavior, in conjunction with how we're using it. However, a user can still scroll the window to read the text. Some of the sizing and text spacing was adjusted slightly to narrow the range of opportunities when this occurs.

Bonus: Vertical alignment of the pointer element has now been adjusted to better align with the icon.

<img width="572" alt="Screen Shot 2021-12-08 at 11 57 11 AM" src="https://user-images.githubusercontent.com/2553268/145257288-3defd7ed-fe25-4235-9ec9-8b1d968c7e71.png">

ilodidoj reviewed (approved):
@louh All working as expected.  Please adjust for the 'No new line at the end of the file' on git file changes msg  for the files below and then merge

crt_portal/static/img/error-blue.svg
crt_portal/static/img/usa-icons/error.svg

louh commented:
(copied & lightly edited from Teams message.)

That's not an error message, that's just letting you know there's not an extra newline there. A lot of our files will not have them.

The reason why this is sometimes important to know is if we have a workflow where some files must be concatenated together, and preserving the line break between file contents is necessary. We do not have that workflow and we don't enforce newlines at end of files, so this doesn't have any impact on us currently.
